File: Nderitu-Dennis/Fizzbuzz.java

public class Fizzbuzz{
    public static void main (String args[]){
        for(int i=1; i<=100; i++){
            if (i%5==0 && i%3==0)
            System.out.println(i + ": FizzBuzz");
            else if (i%5==0)
            System.out.println(i + ": Buzz");
            else if (i%3==0)
            System.out.println(i + ": Fizz");
            else{
                System.out.println(i);
            }

        }


    }

}

File: r-owino/dsa-clinic/src/entrance-challenge/src/fizzbuzz.rs
/* The FizzBuzz problem */

pub struct Solution;

impl Solution {
    pub fn fizzbuzz() -> Vec<String> {
        let mut res = Vec::new();

        for i in 1..=100 {
            if i % 3 == 0 && i % 5 == 0 {
                res.push("FizzBuzz".to_string());
            } else if i % 3 == 0 {
                res.push("Fizz".to_string());
            } else if i % 5 == 0 {
                res.push("Buzz".to_string());
            } else {
                res.push(i.to_string())
            }
        }
        res
    }
}
